Cargo.toml

```diff
@@ -49,6 +49,7 @@ wasmtime-wasi-c = { path = "wasmtime-wasi-c" }
 winapi = "0.3"
 
 [workspace]
+members = ["./wasmtime-*"]
```

Comment:
`wasmtime` doesn't import `wasmtime-hostmodule`, so we have to do this. I would be in favour of making the `wasmtime` root crate reexport `hostmodule` but I wanted to get some feedback first. I actually mention this in the first comment, although I wasn't clear enough about it:

> I haven't added any code that reexports this from the wasmtime root crate but I could imagine that that would be helpful (plus, it would mean we can remove the workspace glob that I added in 1a59141).